### DISCUSSION TITLE
Is it possible to use this to sync only a subfolder of the vault?

### DISCUSSION THREAD

**Discussion by seatrout** [Vinzent03/obsidian-git#529] [Q&A] [answered]
Use case: 
I want to use git to track changes in the drafts of a book I am writing. The book has its own vault in Obsidian. I don' t need git to track all the supporting materials; just the stuff in one subfolder. Is it possible to set the plugin up to do this? It just seems a waste to upload a couple of thousand files, when I only want to track the changes to 20 or 30.

**Comment by Vinzent03** [comment, marked as answer]
Yes that's possible. Use the "Custom base path" setting in the plugin settings under "Advanced".

**Reply by seatrout** [reply]
Thank you! I will get on with this.

**Reply by berot3** [reply]
Mark it as answered?

**Reply by freenandes** [reply]
I don't understand how to use this properly. I always get the "can't find a valid git repo" error. If the subfolder I want to sync is `content` then how would the path look like in the Custom base path field? Like this (it's giving an error)?
```
https://github.com/username/notes/tree/main/content
```

**Reply by freenandes** [reply]
Also, does this path need to be configured before or after the initial clone and setup (in a new device)?

**Reply by Vinzent03** [reply]
You cannot clone only specific directories of a repository. The custom base path setting is used to choose a subdirectory in your vault to be the local repository, which you want to operate on.

**Reply by freenandes** [reply]
Ah! Got it! Thanks! I got lost in translation thinking this would be the option to do that. Now I know...

But would be a nice feat! On desktop I can see a clean folder tree, in the mobile app I see all the git clutter
